yaml/automation/valetudo-notifications.yaml

```diff
@@ -124,6 +124,16 @@ variables:
       success_msg: "Die Reinigung wurde um {{ now().strftime('%-H:%M') }} Uhr erfolgreich abgeschlossen."
       error_msg: "Der Staubsauger hat ein Problem: {{ states( error_message_sensor ) }}"
       default_msg: Alles gut.
+    Francais:
```

Comment:
The language names should be in English. You also need to add them to the language selection parameter list.